alts/src/main/java/io/grpc/alts/GoogleDefaultChannelBuilder.java

```diff
@@ -70,25 +82,23 @@ public static GoogleDefaultChannelBuilder forAddress(String name, int port) {
     return forTarget(GrpcUtil.authorityFromHostAndPort(name, port));
   }
 
-  private GoogleDefaultChannelBuilder(String target) {
-    delegate = NettyChannelBuilder.forTarget(target);
-    InternalNettyChannelBuilder.setDynamicTransportParamsFactory(delegate(), tcpfFactory);
-  }
-
   @Override
   protected NettyChannelBuilder delegate() {
     return delegate;
   }
 
   @Override
   public ManagedChannel build() {
+    @Nullable CallCredentials credentials = null;
+    Status status = Status.OK;
     try {
-      GoogleDefaultInterceptor defaultInterceptor =
-          new GoogleDefaultInterceptor(GoogleCredentials.getApplicationDefault());
-      return delegate().intercept(defaultInterceptor).build();
+      credentials = MoreCallCredentials.from(GoogleCredentials.getApplicationDefault());
     } catch (IOException e) {
-      throw new RuntimeException(e);
+      String message = "Failed to get Google default credentials: " + e.getMessage();
+      logger.log(Level.WARNING, message);
```

Comment:
As a general rule, log an error or propagate the error, but not both. Remove this log statement.

Reply:
Got it! log statement removed.